network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgServerApi.kt

```diff
@@ -68,6 +73,8 @@ class RealWgServerApi @Inject constructor(
         val userPreferredLocation = netPGeoswitchingRepository.getUserPreferredLocation()
         val registerKeyBody = if (selectedServer != null) {
             RegisterKeyBody(publicKey = publicKey, server = selectedServer)
+        } else if (!netPRemoteFeature.showVpnSettings().isEnabled() && !appBuildConfig.isInternalBuild()) {
+            RegisterKeyBody(publicKey = publicKey, server = "*")
```

Comment:
Do we need this check. I guess I know what you're trying to do, but I don't think it fully achieves the goal due to the previous IF, and if settings are disabled users won't get access to geoswitching anyways